src/modules/home/screens/carousel-component.tsx

```diff
@@ -88,53 +32,63 @@ const Icon = ({ focused, ...props }: IconType) => {
 };
 
 const CarouselComponent: React.FC = () => {
-  const navigation = useNavigation<CarouselNavigationProp>();
+  const { data, refetch } = useGetItems();
+
+  useFocusEffect(
+    useCallback(() => {
```

Comment:
I'm not sure how useCallBack is working here. I usually use this hook to memoize a function in a constant like this:
```
 const onRefetch = useCallback(() => {
    refetch(),
 [refetch])}
 ```
 But maybe I'm missing something, please let me know.

Reply:
I'm using it here because in the documentation about the useFocusEffect it's said that the parameter passed should be wrapped in a useCallback to avoid running the effect too often.